src/main/webapp/app/complaints/complaints-for-students/complaints-student-view.component.ts

```diff
@@ -40,6 +41,8 @@ export class ComplaintsStudentViewComponent implements OnInit {
     // flag to indicate exam test run. Default set to false.
     @Input() testRun = false;
```

Comment:
Here you could potentially convert the inputs to the newer input, but I guess that's not really related to the bugfix itself